pages/about.js

```diff
@@ -7,14 +7,253 @@ import { prefixLink } from 'gatsby-helpers';
+            		</div>
+	     					<p>
+	     						UC Berkeley Model United Nations (UCBMUN) engages Berkeley students in debate and discourse on contemporary and historical topics relevant
+			            to international relations, diplomacy, and the United Nations. Students act as delegates representing a country,
+			            NGO, or historical figure as they work to solve international issues through a combination of debate and negotiation. 
+			            Members of UCBMUN are able to improve upon their public speaking, creative problem solving, and interpersonal skills
+			            while learning about a wide range of historical and current foreign affairs. UCBMUN is the official intercollegiate 
+			            team that represents UC Berkeley at eight conferences across the US annually; this travel team is one of the most 
+			            competitive and highest ranked teams in North America. In addition, UCBMUN hosts an annual collegiate conference 
+			            in San Francisco every March, attended by 600 delegates from 45 schools around the world.
+          			</p>
+	     		</div>
+	     	</div>
+
+	     	<div className="col col--1">
+	     		Becoming a Member Organization
+	     	</div>
+	     		<div className="col col--2">
+	     			<div className="col--2__section">
+	     					IRC membership is open to all registered student organizations (RSOs) at Berkeley who focus on international affairs 
+		            and seek to foster cooperation among international relations organizations on campus. 
+		            Organizations interested in joining the IRC must submit a formal statement of interest to the Secretary-General
+		            detailing their international relations focus, their interest and level of involvement in the IRC.  Please 
+		            reach out to ircofberkeley@gmail.com to submit a statement of interest, or with any questions regarding joining the IRC! 
+	     			</div>
+	     		</div>
+
+	     	<div className="col col--1">
+        	Leadership
+        </div>
+        <div className="table">
+          <div className="desc">
+            <img src="/img/spandi.jpg" />
+            <hr></hr>
+            <h4>Secretary-General</h4>
+            <hr></hr>
+            <p>
+              <b>Spandana (Spandi) Singh</b> is a fourth year at Cal double majoring in Development Studies and Media Studies,
+              minoring in Global Poverty & Practice and completing the Certificate in Technology Entrepreneurship.
+              Originally from India, Spandi's interests include global health, financial inclusion, entrepreneurship
+              and tech policy. Spandi has previously interned at UNICEF and the World Bank Group, and is a co-founder 
+              of two social enterprises, Sira Corporation and REcharge Initiative. She is currently a Public Policy Fellow 
+              at Twitter in San Francisco.
+            </p>
+          </div>
+        </div>
+
+	    <div className="table">
```

Comment:
Instead of using ```table```, are you able to also use similar styles from base.scss? For example, using ```col``` and ```col--1``` and ```col--2``` instead